### PR TITLE
[WIP] Extend prose documentation for round 2 Lookit test users

### PR DESCRIPTION
**Pull request by kimberscott**
## Purpose

Provide additional documentation for new users writing their own studies on Lookit in "round 2" of testing.

## Summary of changes

Changes should be confined to docs/source/ and it should be really hard for them to break anything.

### CODE REVIEW

File: docs/source/stimuli.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+For basic editing of audio files, if you don't already have a system in place, we highly recommend [Audacity](http://www.audacityteam.org/). You can create many "tracks" or select portions of a longer recording using labels, and export them all at once; you can easily adjust volume so it's similar across your stimuli; and the simple "noise reduction" filter works well.
+
+### File formats
+
+To have your media play properly across various web browsers, you will generally need to provide multiple file formats. For a comprehensive overview of this topic, see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats). 
+
+MIT's standard practice is to provide mp3 and ogg formats for video, and webm and mp4 (H.264 video codec + AAC audio codec) for video, to cover modern browsers. The easiest way to create the appropriate files, especially if you have a lot to convert, is to use the command-line tool [ffmpeg](https://ffmpeg.org/). It's a bit of a pain to get used to, but then you can do almost anything you might want to with audio and video files.
```
Comment (abought) [inline docs/source/stimuli.md]: Should that say MP3 "for audio"?

**Review by abought — APPROVED** [review]
I see this is a WIP, but would see no issues with merging this when ready.

File: docs/source/stimuli.md (hunk 1 above)
Comment (kimberscott) [reply docs/source/stimuli.md]: Yes--thanks! (And if you ever get tired of software development, seems like copyediting would be a pretty safe fallback... :) )

Comment (abought) [reply docs/source/stimuli.md]: Sorry, old habit from my grad school days. :) 

(there actually are copyediting agencies for the sciences, but from what I hear they don't pay the bills nearly as much as you'd think....)

**Pull request merged**